src/NodeJS/Server.php

```diff
@@ -46,6 +46,11 @@
     protected $threshold;
 
     /**
+     * @var array
+     */
+    protected $options;
```

Comment:
@stof I tried to follow what's already there and all the other properties are protected while having setters and getters.